test/v2/services/RetrieveUkPropertyBsasServiceSpec.scala

```diff
@@ -36,7 +36,7 @@ class RetrieveUkPropertyBsasServiceSpec extends UnitSpec{
   private val nino = Nino("AA123456A")
   val id = "f2fb30e5-4ab6-4a29-b3c1-c7264259ff1c"
   val adjustedStatus = Some("03")
-  private val correlationId = "X-123"
+  private implicit val correlationId = "X-123"
```

Comment:
Please remove once you have added the **implicit val correlationId: String = "X-123"** to the ServiceSpec and this class extends it instead (Change from UnitSpec to ServiceSpec)